stack/reactive-resume.yml

```diff
@@ -0,0 +1,38 @@
+version: "2"
+services:
+  proxy:
+    image: martadinata666/nginx:mainline
+    ports:
+      - ${PORT}:80
+    volumes:
+      - /portainer/Files/AppData/Config/reactive-resume/nginx.conf:/etc/nginx/conf.d/default.conf
+    restart: unless-stopped
+
+  app:
+    image: martadinata666/reactive-resume:v3.4.6
+    restart: unless-stopped
+    environment:
+      - SECRET_KEY=${SECRET_KEY}
+      - POSTGRES_HOST=db
+      - POSTGRES_PORT=5432
+      - POSTGRES_USER=${DATABASE_USERNAME}
+      - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
+      - POSTGRES_DB=reactiveresume
+      - JWT_SECRET=${JWT_SECRET}
+      - GOOGLE_API_KEY=${GOOGLE_API_KEY}
+      - PUBLIC_FLAG_DISABLE_SIGNUPS=${DISABLE_SIGNUPS}
+      - STORAGE_S3_ENABLED=${ENABLE_S3_STORAGE}
```

Comment:
If you don't want to expose `ENABLE_S3_STORAGE`, then change it here to `false` and leave it for the users to change it directly on Portainer if they want it to

Reply:
Ah i see, just this commenting timeline pretty confusing, i asked same thing up there